package.json

```diff
@@ -139,12 +153,20 @@
                     "when": "view == scm-local-branches && viewItem == activeBranch"
                 },
                 {
-                    "command": "scm-local-branches.branch.delete-upstream",
+                    "command": "scm-local-branches.branch.unset-upstream",
                     "when": "view == scm-local-branches && viewItem == branch"
                 },
                 {
-                    "command": "scm-local-branches.branch.delete-upstream",
+                    "command": "scm-local-branches.branch.unset-upstream",
                     "when": "view == scm-local-branches && viewItem == activeBranch"
+                },
+                {
+                    "command": "scm-local-branches.branch.fetch",
+                    "when": "view == scm-local-branches && viewItem == branch"
+                },
+                {
```

Comment:
Does checkout via the git extension not already fetch? It seems like it's usually up to date for me already.

Reply:
I've checked: checkout does only that. It does not fetch changes nor pull